src/Chainweb/BlockHeader.hs

```diff
@@ -33,8 +33,17 @@
 --
 module Chainweb.BlockHeader
 (
+-- * Newtype wrappers for function parameters
+  ParentHeader(..)
+, ParentCreationTime(..)
```

Comment:
There are likely more places such a type would be useful. At one point I had a `PrevBlock` type in `Coordination.hs`. Your type here should take precedence.

Reply:
Yeah, I didn't want to clutter this PR. But the intention is to use this in other places, too.